pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -143,6 +145,13 @@ pub mod pallet {
 			+ IsType<<Self as frame_system::Config>::RuntimeCall>;
 
 		type Capacity: Nontransferable;
+		type CapacityEligibleCalls: Contains<<Self as Config>::RuntimeCall>;
```

Comment:
Accepting alternate names

Reply:
How about CapacityWhitelistedCalls?

Reply:
🤔 not that much different...let's sit with it and change it later if we decide we don't like it.